src/message/messageActionSheet.js

```diff
@@ -112,7 +112,7 @@ muteTopic.title = 'Mute topic';
 muteTopic.errorMessage = 'Failed to mute topic';
 
 const deleteTopic = async ({ auth, message, dispatch, ownEmail, _ }) => {
-  const alertTitle = _("Are you sure you want to delete the topic '{topic}'?", {
+  const alertTitle = _('Are you sure you want to delete the topic ’{topic}’?', {
```

Comment:
Hmm! So I actually didn't think about the case when these are really quotation marks -- I was thinking of the apostrophes. This doesn't look right to me:
> `to delete the topic ’{topic}’`

and I think the issue is that U+2019 is the *right* single quotation mark. There's also ‘ U+2018 LEFT SINGLE QUOTATION MARK.

Looking afresh at this particular message: it seems like the normal thing in English text that isn't code would be to use double-quotes, anyway. Could use either the plain ASCII `"`, or the Unicode quotation marks:
“ U+201C LEFT DOUBLE QUOTATION MARK
” U+201D RIGHT DOUBLE QUOTATION MARK
What do you think?

Reply:
Sure, sounds good!